systemtest/src/test/java/io/strimzi/systemtest/rollingupdate/KafkaRollerST.java

```diff
@@ -168,11 +171,15 @@ void testKafkaPodCrashLooping() {
     void testKafkaPodImagePullBackOff() {
         KafkaResource.kafkaPersistent(CLUSTER_NAME, 3, 3).done();
 
-        KafkaResource.replaceKafkaResource(CLUSTER_NAME, kafka ->
-                kafka.getSpec().getKafka().setImage("strimzi/kafka:not-existent-tag"));
+        KafkaResource.replaceKafkaResource(CLUSTER_NAME, kafka -> {
+            kafka.getSpec().getKafka().setImage("strimzi/kafka:not-existent-tag");
+            kafka.getSpec().getZookeeper().setImage("strimzi/kafka:latest-kafka-" + Environment.ST_KAFKA_VERSION);
```

Comment:
What's the purpose of this?

Reply:
When you set image just to Kafka and not the Zookeeper -> so the ZK will not have any image -> he will get the Kafka image. So this is a test where the Kafka pod should go to `ImagePullBackOff` not the ZK. Setting the Zookeeper image will make that just Kafka pod will "fail". We discussed this with @scholzj and @stanlyDoge.

Reply:
Ok, thanks for the explanation